modal-go/sandbox_filesystem.go

```diff
@@ -12,14 +12,14 @@ import (
 type SandboxFile struct {
 	fileDescriptor string
 	taskId         string
-	ctx            context.Context
+	cpClient       pb.ModalClientClient
 }
 
 // Read reads up to len(p) bytes from the file into p.
 // It returns the number of bytes read and any error encountered.
 func (f *SandboxFile) Read(p []byte) (int, error) {
 	nBytes := uint32(len(p))
-	totalRead, _, err := runFilesystemExec(f.ctx, pb.ContainerFilesystemExecRequest_builder{
+	totalRead, _, err := runFilesystemExec(context.Background(), f.cpClient, pb.ContainerFilesystemExecRequest_builder{
```

Comment:
These methods can't take request scoped Contexts since the io.* interfaces don't have it, so falling back to using Background().